PresentScreenings/Controllers/ScreeningDialogController.cs

```diff
@@ -47,7 +47,7 @@ public ViewController Presentor
 
         #region Interface Implementation Properties
         public Screening CurrentScreening => Presentor.Plan.CurrScreening;
-        public List<Screening> Screenings => ViewController.FilmScreenings(Presentor.Plan.CurrScreening.FilmId);
+        public List<Screening> Screenings => Presentor.Plan.CurrScreening.Film.FilmScreenings;
```

Comment:
Please consider to add a `FilmScreenings` property to `Screening` too, as to prevent two queries on the screenings list.

Reply:
Done.